tasks/notify.py

```diff
@@ -436,11 +436,39 @@ def update_statistics(job_executions: PipelineRuns):
 
 
 def send_notification(ctx: Context, alert_jobs):
+    """
+    Send alert notifications to slack and send metrics to Datadog
+    """
+    metrics = []
+    timestamp = int(datetime.now(timezone.utc).timestamp())
+
     message = alert_jobs["consecutive"].message(ctx) + alert_jobs["cumulative"].message()
     message = message.strip()
 
     if message:
         send_slack_message("#agent-platform-ops", message)
+        print('Sent message')
+
+        # TODO : When these notifications will be sent to many teams, update the team tag (https://datadoghq.atlassian.net/browse/ACIX-255)
+        owner = '@datadog/agent-all'
+        team_name = owner.removeprefix('@datadog/')
+
+        # Create metrics for consecutive and cumulative alerts
```

Comment:
What will be the objective of this metric? Will we put it in a dashboard or a monitor?

Reply:
It is to monitor how many notifications are sent to the teams, I've added a metric with the current notification system. We will be able to compare them